src/features/Instructors/InstructorsPage/index.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState, useReducer } from 'react';
+
+import { logError } from '@edx/frontend-platform/logging';
+import Container from '@edx/paragon/dist/Container';
+import {
+  Pagination,
+} from '@edx/paragon';
+import InstructorsTable from '../InstructorsTable';
+
+import { getInstructorData } from '../data/api';
+import reducer from './reducer';
+import {
+  FETCH_INSTRUCTOR_DATA_REQUEST,
+  FETCH_INSTRUCTOR_DATA_SUCCESS,
+  FETCH_INSTRUCTOR_DATA_FAILURE,
+  UPDATE_CURRENT_PAGE,
+} from '../actionTypes';
+import { RequestStatus } from '../../constants';
+
+const initialState = {
+  data: [],
+  status: RequestStatus.SUCCESS,
+  error: null,
+  currentPage: 1,
+  numPages: 0,
+};
+
+const InstructorsPage = () => {
+  const [state, dispatch] = useReducer(reducer, initialState);
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const fetchData = async () => {
+    dispatch({ type: FETCH_INSTRUCTOR_DATA_REQUEST });
+
+    try {
+      const response = await getInstructorData(currentPage);
```

Comment:
Could you please use CamelCaseobject? https://github.com/openedx/frontend-platform/blob/master/src/utils.js#L73. An example of use is https://github.com/Pearson-Advance/frontend-app-institution-portal/blob/master/src/features/Students/StudentsPage/index.jsx#L99.